Admin/Admin.php

```diff
@@ -2964,6 +2964,27 @@ public function checkAccess($action, $object = null)
     /**
      * {@inheritdoc}
      */
+    public function hasAccess($action, $object = null)
```

Comment:
You're right, @tim96 should rebase on the tip of master, and it will disappear from the diff.